src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentPanelMobile.js

```diff
@@ -20,7 +20,7 @@ class DocumentPanelMobile extends Component {
     const { documentDetails: doc, isLoading, loansInfo } = this.props;
     return (
       <div
-        className="document-panel"
+        className="document-series-panel"
```

Comment:
is this intentional ? seems like a mismatch between naming of the component and the class